src/qonlinetts.h

```diff
@@ -184,15 +220,43 @@ class QOnlineTts : public QObject
      */
     static Voice voice(const QString &voiceCode);
 
+    /**
+     * @brief Voice region from code
+     *
+     * Used only by Google
+     *
+     * @param regionCode region code
+     * @return corresponding region code
+     */
+    static Region region(const QString &regionCode);
+
+    static QString regionName(Region region);
+
+    /**
+     * @brief Voice region preferences
+     * @return voice region preferences
+     */
+    const QMap<QOnlineTranslator::Language, Region> &regionPreferences() const;
+
+    /**
+     * @brief Set voice region preferences
+     * @param region preferences
+     */
+    void setRegionPreferences(const QMap<QOnlineTranslator::Language, Region> &regionPreferences);
```

Comment:
How about to make it a little shorter? The same for getter.
```suggestion
    void setRegions(const QMap<QOnlineTranslator::Language, Region> &regionPreferences);
```